step.yml

```diff
@@ -37,37 +37,62 @@ inputs:
       title: "Platform filter"
       description: |
         Filters which platform to build.
-        Leave it empty to build both android and ios.
-      value_options:
-      - "ios"
-      - "android"
-      - "ios,android"
+        Comma separated list of platforms to build.
+
+        Available platforms:
+        * ios
+        * android
+        * mac
+
+        Format examples:
+        * mac
+        * android
+        * ios,android,mac
   - export_options_path:
     opts:
       title: "Export options path"
       description: |-
-        Used for Xcode version 7 and above.
+        [!] Used for iOS & OS X projects.
 
         Specifies a path to a plist file that configures archive exporting.
-        If empty, step generates these options based on provisioning profile,
-        with default values.
+        If empty, step generates these options.
 
-        Auto generated export options available for export methods:
+        Auto generated export options available for iOS export methods:
         - app-store
         - ad-hoc
         - enterprise
         - development
 
-        If step doesn't find export method based on provisioning profile, development will be use.
+        See `xcodebuild -help` for more information.
+  - export_method: "development"
+    opts:
+      title: "Export method"
+      description: |-
+        [!] Used for OS X projects.
```

Comment:
Remove [!] from description. Instead we should add `Export method - use only for OS X projectects`